IntoHistory/NMapViewController.swift

```diff
@@ -11,21 +11,105 @@ import NMapsMap
 	
 class NMapViewController: UIViewController, CLLocationManagerDelegate {
 
+    // MARK: - Property
+    
     private let courseJSONLoader = LoadingCourseJSON().courses
     var locationManager = CLLocationManager()
+    let infoWindow = NMFInfoWindow()
+    let dataSource = NMFInfoWindowDefaultTextSource.data()
+    
+    // MARK: - View
+    
+    let hStackView: UIStackView = {
```

Comment:
p1;

접근제어자 추가 부탁드립니다!
```suggestion
    private let hStackView: UIStackView = {
```

Reply:
> p1;
> 
> 접근제어자 추가 부탁드립니다!

6c480ff
수정하였습니다.